taxcalc/calculate.py

```diff
@@ -160,6 +160,36 @@ def increment_year(self):
         self.policy.set_year(self.policy.current_year + 1)
         self.behavior.set_year(self.policy.current_year)
 
+    def year_increment(self, advance_to_year):
+        '''
+        The year_increment function gives an optional way of implementing
+        increment year functionality by immediately specifying the year
+        as input. Robustness has been added to ensure input is of good shape.
+        '''
+        if isinstance(advance_to_year, int):
+            iteration = advance_to_year - self.records.current_year
+            if iteration > 0:
+                if iteration > 3:
+                    for i in range(iteration):
+                        self.increment_year()
+                    assert self.records.current_year == advance_to_year
+                    print("Your data and policy have been further extrapolated"
+                          " to " + str(self.records.current_year) + ".")
+                else:
+                    for i in range(iteration):
+                        self.increment_year()
+                    assert self.records.current_year == advance_to_year
+                    print("Your data have been further extrapolated to " +
+                          str(self.records.current_year) + ".")
+            elif iteration == 0:
+                print("You are already in " +
+                      str(self.records.current_year) + ".")
+            else:
+                print(str(self.records.current_year) + " is the earliest " +
```

Comment:
We should raise (exceptions)[https://docs.python.org/2/tutorial/errors.html] on bad input. I would suggest a ValueError.